adrs/modules/20230816-iam-refactor.md

```diff
@@ -202,6 +218,13 @@ variable "iam_bindings_additive" {
   nullable = false
 }
 
+variable "iam_by_principals_additive" {
+  description = "Additive IAM binding in {PRINCIPAL => [ROLES]} format. Principals need to be statically defined to avoid cycle errors. Merged internally with the `iam_bindings_additive` variable."
```

Comment:
It's actually not a cycle errors, but "for_each will be known only after apply" error :-)

Reply:
Done. But we still have a lot of places where it shows up.